### PR TITLE
First pass at styling

### PR DESCRIPTION
**Pull request by creuter**
triangle ember

### CODE REVIEW

File: source/assets/stylesheets/base/_typography.scss [hunk 1]
```diff
@@ -9,25 +9,35 @@ body {
 
 h1, h2, h3, h4, h5, h6 {
   font-family: $header-font-family;
```
Comment (kylefiedler) [inline source/assets/stylesheets/base/_typography.scss]: Don't think you need this line anymore

File: source/assets/stylesheets/base/_variables.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Typography
-$sans-serif: $helvetica;
+$sans-serif: "ff-tisa-sans-web-pro", $helvetica;
```
Comment (kylefiedler) [inline source/assets/stylesheets/base/_variables.scss:2]: :heart: Tisa

File: source/assets/stylesheets/base/_variables.scss [hunk 3]
```diff
@@ -14,27 +14,33 @@ $base-border-radius: em(3);
...
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
+$light-font-color: lighten($base-font-color, 15%);
+$base-accent-color: $orange;
 
 // Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
+$base-link-color: $blue;
+$hover-link-color: darken($base-link-color, 15%);
```
Comment (kylefiedler) [inline source/assets/stylesheets/base/_variables.scss:34]: You and your damn `%`

**Comment by kylefiedler** [comment]
Delete that one line and you've got yourself an :+1:

File: source/assets/stylesheets/base/_variables.scss (hunk 3 above)
Comment (creuter) [inline source/assets/stylesheets/base/_variables.scss:34]: I'm waging a very slow war.

**Pull request merged**